lifter-data/social-instagram.csv

```diff
@@ -382,7 +382,7 @@ Ed Knoblock,ed.k52
 Edan Davey,edandavey
 Eddie Berglund,strongeddie
 Eduard Khanjyan,eduardkhanjyan
-Eduardo Garcia,kimbo_halfslice_
+Eduardo García,kimbo_halfslice_
```

Comment:
I think this is an error -- there's one lifter named "Eduardo Garcia" who competes USPA, and one named "Eduardo García" who competes FEMEPO.

We don't have a way to handle disambiguation when one uses accents and the other doesn't. For the moment, it may be best to keep them as Eduardo Garcia.

The other name corrections are probably multiple lifters under one name also.